build.gradle.kts

```diff
@@ -4,6 +4,16 @@ import org.gradle.api.tasks.testing.logging.TestLogEvent
 
 // Top-level build configuration
 
+buildscript {
+    repositories {
+        google()
+    }
+
+    dependencies {
+        classpath("com.android.tools.build:gradle:${Versions.androidGradlePlugin}")
```

Comment:
Thanks for the tip, I'll look into this!

Reply:
Done, for some weird reason I couldn't use a `when` statement... 🤔  so I ended up using an if/else.

Reply:
Oh right, we're still using Groovy rather than Kotlin for `settings.gradle`. Forgot to convert that one over to Kotlin when we did the other gradle files.